src/v3/src/types/schema.ts

```diff
@@ -39,6 +39,8 @@ export type FormBag = {
   }
 };
 
+export type IdxMessageWithName = IdxMessage & { name?: string | undefined; };
```

Comment:
Totally right about the type, updated.

> Is this a type fix that could/should be made by auth-js?

This `name` field is added by SIW v3, it was only added for usage by the password confirmation transformer because we needed a way to decipher between the new password field and confirm password field errors. I don't think this should be modified by auth-js for that reason: 

This is where we use the `name` field: https://github.com/okta/okta-signin-widget/blob/0.0/src/v3/src/transformer/password/transformEnrollPasswordAuthenticator.ts#L83